sunpy/net/dataretriever/sources/eve.py

```diff
@@ -0,0 +1,105 @@
+# Author: Rishabh Sharma <rishabh.sharma.gunner@gmail.com>
+# This module was developed under funding by
+# Google Summer of Code 2014
+
+from __future__ import absolute_import, division, print_function
+
+from sunpy.extern.six.moves.urllib.parse import urljoin
+
+from ..client import GenericClient
+
+
+__all__ = ['EVEClient']
+
+
+class EVEClient(GenericClient):
```

Comment:
In fact, VSO supports downloading many eve products so this would only need to support ones not in the vso